test/commands/create.test.ts

```diff
@@ -131,7 +133,15 @@ describe('org:create:user', () => {
     if (readsFile) {
       $$.SANDBOXES.CONNECTION.stub(Connection.prototype, 'singleRecordQuery').resolves({ Id: '12345678' });
       $$.SANDBOX.stub(Logger.prototype, 'debug');
-      $$.SANDBOX.stub(fs.promises, 'readFile').resolves(JSON.stringify(readsFile));
+      if (typeof readsFile !== 'boolean') {
+        const fsStub = $$.SANDBOX.stub(fs.promises, 'readFile');
+        fsStub.withArgs('parent/child/file.json').resolves(JSON.stringify(readsFile));
```

Comment:
only changes readFile for the file it needs to stub.  callThrough lets the original implementation work in other cases.